### PR TITLE
Expand the manual testing section

### LINKED ISSUE #41: Manual testing
Extend the part about manual testing.

### PR DESCRIPTION
**Pull request by abelsiqueira**
Close #41

### CODE REVIEW

File: overview.tex [hunk 1]
```diff
@@ -262,12 +262,23 @@ \subsection*{Quality control}
...
+    graphics. This script is also run on Travis CI, however, we can only verify
+    that it outputs the desired graphics locally.
+
+    This script uses artifial solver information accessible using \verb+--demo+
+    as argument in the perprof-py call.
+    For instance, to test that the Tikz installation is successful, one can run
+\begin{verbatim}
+perprof-py --demo -o /tmp/tmp --tikz
+\end{verbatim}
+    If everything is correct, this will generate file /tmp/tmp.pdf with an
```
Comment (rgaiacs) [inline overview.tex]: `/tmp/tmp.pdf` -> `\verb+/tmp/tmp.pdf+`

**Comment by abelsiqueira** [comment]
Fixed.

File: overview.tex [hunk 2]
```diff
@@ -262,12 +262,24 @@ \subsection*{Quality control}
...
-    These tests use artificial solver information accessible using \verb+--demo+
-    as argument in the perprof-py call. The user can run a script that makes
-    several of these graphics in various formats by entering the folder
+    In addition we also run a script to generate several performance profile
+    graphics. This script is also run on Travis CI, however, we can only verify
+    that it outputs the desired graphics locally.
+
+    This script uses artifial solver information accessible using \verb+--demo+
+    as argument in the perprof-py call.
+    For instance, to test that the Tikz installation is successful, one can run
```
Comment (rgaiacs) [inline overview.tex]: Tikz -> TikZ

Sorry to missed it at the first time.

**Comment by abelsiqueira** [comment]
Done

**Pull request merged**